### PR TITLE
collab. personal touch on the README.md

### PR DESCRIPTION
**Pull request by newls**
everything in the PR title I guess

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
-# Example Plugin for GLPI
+# GLPi Example Plugin
 
 ## Introduction
 
-This plugin aims to introduce how to interact with glpi HOOKS.  
-Most of implemented features are for example and do nothing.  
- 
-See [documentation](https://github.com/pluginsGLPI/example/wiki) for plugin creation
```
Comment (orthagh) [inline README.md]: This part seems important.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+This plugin implements a few basic glpi hooks.
+Most of implemented features stands for example and are not
+expected to provide fantastic functionality.
+
```
Comment (newls) [inline README.md]: Personal touch, I tried not to change the meaning, PR still can be rejected.
I think the developer implements available hooks in a way, I wanted to put an emphasis on it.

**Comment by newls** [comment]
Personal touch, I tried not to change the meaning, PR still can be rejected.
I think the developer implements available hooks in a way, I wanted to put an emphasis on it.

"and do nothing" => "and are not expected to provide fantastic functionality other than teaching ?."
I don't know, PR still can be rejected or discussed in a way if you want to.

**Comment by orthagh** [comment]
No i mentionned just this part : "See [documentation](https://github.com/pluginsGLPI/example/wiki) for plugin creation"

**Pull request merged**